src/main/java/com/palantir/gradle/versions/ConflictSafeLockFile.java

```diff
@@ -35,7 +35,8 @@
 import org.gradle.api.GradleException;
 
 final class ConflictSafeLockFile {
-    private static final String HEADER_COMMENT = "# Run ./gradlew writeVersionsLocks to regenerate this file";
+    private static final String HEADER_COMMENT =
```

Comment:
will probably break dependabot 😅

Reply:
We checked, [renovate-bot should be fine as it just matches the start of the header line](https://github.com/renovatebot/renovate/blob/f94c2d3ce7a86f3ef168d5db118bbea41def573e/lib/modules/manager/gradle/extract/consistent-versions-plugin.ts#L12).